huaweicloud/config/endpoints.go

```diff
@@ -158,10 +160,11 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		WithOutProjectID: true,
 		Product:          "CCE",
 	},
-	"aom": {
-		Name:    "aom",
-		Version: "svcstg/icmgr/v1",
-		Product: "AOM",
+	"swr": {
+		Name:             "swr-api",
+		Version:          "v2",
+		WithOutProjectID: true,
+		Product:          "SWR",
```

Comment:
It seems that this modification does not belong to this commit.

Reply:
only adjust the order, I think it's ok